third_party/WebKit/Source/modules/webcl/DOMWindowWebCL.h

```diff
@@ -6,11 +6,11 @@
 #define DOMWindowWebCL_h
 
 #include "core/frame/DOMWindowProperty.h"
-#include "core/frame/LocalDOMWindow.h"
```

Comment:
This line originally exists in `DOMWindowWebCL.cpp`. Though the header has already been indirectly included there, I suggest to add it back for better readability.